packages/jasmine/jasmine_node_test.bzl

```diff
@@ -62,10 +62,10 @@ def jasmine_node_test(
         expected_exit_code = 0,
         tags = [],
         config_file = None,
-        # Replaced by pkg_npm with jasmine = "@npm//@bazel/jasmine",
-        jasmine = "@npm_bazel_jasmine//:jasmine__pkg",
+        # Replaced by pkg_npm with jasmine = "//@bazel/jasmine",
```

Comment:
Isn't it replaced with `@npm//@bazel/jasmine`?

Reply:
nope, we don't want to hard-code that the user names their repository "@npm" because it only works in the simple case.

The rule will execute in the `@npm_whatever` repo so it shouldn't be fully-qualified